packages/design-tokens/package.json

```diff
@@ -51,6 +51,7 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@metamask/design-system-react": "workspace:^",
```

Comment:
We are using the `design-system-react` `Text` component in stories instead of duplicating components